src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -270,14 +271,22 @@ object MavenArtifactId {
   }
 }
 
-case class MavenCoordinate(group: MavenGroup, artifact: MavenArtifactId, version: Version) {
-  def unversioned: UnversionedCoordinate = UnversionedCoordinate(group, artifact)
-  def asString: String = s"${group.asString}:${artifact.asString}:${version.asString}"
+case class MavenCoordinate(group: MavenGroup, artifact: MavenArtifactId, classifier: Classifier, version: Version) {
+  def unversioned: UnversionedCoordinate = UnversionedCoordinate(group, artifact, classifierOpt())
+  def asString: String = classifier match {
+    case Classifier("") => s"${group.asString}:${artifact.asString}:${version.asString}"
+    case Classifier(c) => s"${group.asString}:${artifact.asString}:jar:${c}:${version.asString}"
```

Comment:
can you comment why we always want `jar` in there? I don't know much about classifiers, so I would have thought that would have other types in there.

Reply:
We definitely don't always want `jar` here; ideally we'd allow specifying the packaging, but I was a little overwhelmed while trying to learn Scala and make this work simultaneously. :) 

`jar` is assumed to be the default packaging when none is specified, so hard-coding jar here was just to maintain backwards-compatibility with what was there before. https://maven.apache.org/pom.html#Maven_Coordinates